File: aula6/cash.py
from cs50 import get_float

coins = [25, 10, 5, 1]

money = -1
while money < 0:
    money = get_float("Change owed: ")

cents = int(money * 100)

amount = 0
for coin in coins:
    rest = cents % coin
    amount += cents // coin
    cents = rest
print(amount)

File: aula01/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void)
{
    float money;
    do
    {
        money = get_float("Chance owed: ");
    }
    while (!(money > 0));
    int cents = round(money * 100);
    int sum = 0;
    while (cents != 0)
    {
        if (cents >= 25)
        {
            cents = cents - 25;
            sum++;
        }
        else if (cents >= 10)
        {
            cents = cents - 10;
            sum++;
        }
        else if (cents >= 5)
        {
            cents = cents - 5;
            sum++;
        }
        else
        {
            cents = cents - 1;
            sum++;
        }
    }
    printf("%d\n", sum);
}